resources/css/app.css

```diff
@@ -1,3 +1,21 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import 'tailwindcss';
+
+@theme {
+  --font-family-sans: "Inter Variable", sans-serif;
+}
```

Comment:
Let's use `inter` because [I'm suck at design](https://youtu.be/31pBMi0UdYE?si=9SNY1zPA_Ro2yfBB&t=32)